src/main/java/pl/multiplex/multiplexreservation/dao/entity/Seat.java

```diff
@@ -1,41 +1,108 @@
 package pl.multiplex.multiplexreservation.dao.entity;
 
+import javax.persistence.*;
+import java.util.Set;
+
+import static javax.persistence.GenerationType.SEQUENCE;
+
+@Entity
 public class Seat
 {
+    @Id
+    @SequenceGenerator(
+            name = "seat_sequence",
+            sequenceName = "seat_sequence",
+            allocationSize = 1
+    )
+    @GeneratedValue(
+            strategy = SEQUENCE,
+            generator = "seat_sequence"
+    )
+    @Column(
+            name = "seatId"
+    )
+    private Long seatId;
+    private int nr_row;
+    private int nr_seat;
+    @ManyToOne
+    private CinemaRoom cinemaRoomId;
+    @OneToMany
+    private Set<Reservation> reservations;
```

Comment:
Myślisz, że siedzenie powinno trzymać listę wszystkich powiązanych z nim rezerwacji?
Co się stanie, jeżeli będziemy chcieli zaimplementować operację zmiany miejsca w ramach jednej rezerwacji?

Reply:
Takie przyjąłem założenie podczas projektowania bazy danych. Bardzo możliwe że nie jest to dobre rozwiązanie.
Jak byśmy chcieli zmienić rezerwację na inne siedzenie to by trzeba było:
1. Wejść do siedzenia w aktualnej rezerwacji i usunąć z niego tą rezerwację (o tym reservationId).
2. Zmienić seatId w rezerwacji na nowy.
3. Wejść do nowego siedzenia i dodać mu nową rezerwację o tym reservationId